Tests/SwerverTests/RequestSpec/HttpRequestParserSpec.swift

```diff
@@ -0,0 +1,23 @@
+import Quick
+import Nimble
+import Swerver
+import Foundation
+
+class HttpRequestParserSpec: QuickSpec {
+    override func spec() {
+        describe("An HTTP Request Parser") {
+            var httpRequestParser: HttpRequestParser!
+            
+            beforeEach {
+                httpRequestParser = HttpRequestParser()
+            }
+            
+            it("returns an HttpRequest Object") {
```

Comment:
Style comment: I tend to avoid specifying class/variable/function/method/etc names in test descriptions, instead preferring my tests to read like prose.